hack/setup-test-cluster.sh

```diff
@@ -15,11 +17,11 @@ do
 done
 
 echo "[INFO]: Labeling first node as the ptp grandmaster"
-node=$(${OC_TOOL} get nodes -o name | sed -n 1p)
+node=$(${OC_TOOL} get nodes -o name --selector '!${NON_PTP_LABEL}' | sed -n 1p)
```

Comment:
`'!${NON_PTP_LABEL}'` -> the single quotes prevent using the actual value, please use double quotes. See failure in e2e test.